ivy_tests/test_ivy/test_functional/test_core/test_elementwise.py

```diff
@@ -2483,3 +2483,39 @@ def test_isreal(
         fn_name="isreal",
         x=x[0],
     )
+
+
+# diff
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=1,
+    ),
+    num_positional_args=helpers.num_positional_args(fn_name="diff"),
+)
+def test_diff(
```

Comment:
`test_diff` should be in `test_functional/test_experimental/test_core/test_elementwise.py`

Reply:
moved test_diff to the correct place